test/server/getters/get-markets-awaiting-reporting.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const unlink = require("fs").unlink;
+const join = require("path").join;
```

Comment:
Note: `unlink` and `join` are both unused (I know this is just cruft from other test files, my bad for not removing these everywhere...).

Reply:
removed from all tests